### PR TITLE
Isbm sanitizers fix and unit test

### PR DESCRIPTION
**Pull request by isbm**
### What issues does this PR fix or reference?

Sanitizers allows trailing dots on the hostname.

### Tests written?

Yes

### CODE REVIEW

File: tests/unit/utils/sanitizers_test.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        value = '   ../ ../some/dubious/hostname      '
+        response = clean.hostname(value)
+        assert response == 'somedubioushostname'
+
+    test_sanitized_id = test_sanitized_hostname
+
+
+if __name__ == '__main__':
+    from integration import run_tests
+    run_tests(ArgsTestCase, needs_daemon=False)
```
Comment (cachedout) [inline tests/unit/utils/sanitizers_test.py]: `run_tests(SanitizersTestCase, needs_daemon=False)`

**Review by cachedout — CHANGES_REQUESTED** [review]
Just one small fix at the end of the test case and then should be ready. This is really good! Thanks as always, @isbm

File: tests/unit/utils/sanitizers_test.py (hunk 1 above)
Comment (isbm) [inline tests/unit/utils/sanitizers_test.py]: @cachedout ahhhh, copydamnpaste for scaffolding from ArgsTestCase :laughing: Yessir.

Comment (cachedout) [inline tests/unit/utils/sanitizers_test.py]: It happens. :] Thanks for the quick response as always! Much appreciated.

**Comment by isbm** [comment]
@cachedout done.

**Comment by cachedout** [comment]
Merged. Many thanks!

**Pull request merged**